tests/unit/amo/containers/TestHome.js

```diff
@@ -1,31 +1,104 @@
+import { shallow } from 'enzyme';
 import React from 'react';
-import { findDOMNode } from 'react-dom';
-import {
-  findRenderedComponentWithType,
-  renderIntoDocument,
-} from 'react-addons-test-utils';
-import { Provider } from 'react-redux';
 
-import { HomeBase } from 'amo/containers/Home';
+import {
+  CategoryLink,
+  ExtensionLink,
+  HomeBase,
+  ThemeLink,
+  mapStateToProps,
+} from 'amo/components/Home';
+import Link from 'amo/components/Link';
+import { CLIENT_APP_ANDROID, CLIENT_APP_FIREFOX } from 'core/constants';
 import { dispatchSignInActions } from 'tests/unit/amo/helpers';
 import { getFakeI18nInst } from 'tests/unit/helpers';
 
 
 describe('Home', () => {
-  it('renders a heading', () => {
-    const { store } = dispatchSignInActions();
-
-    const root = findRenderedComponentWithType(renderIntoDocument(
-      <Provider store={store}>
-        <HomeBase dispatch={store.dispatch} i18n={getFakeI18nInst()} />
-      </Provider>
-    ), HomeBase);
-    const rootNode = findDOMNode(root);
-    const content = [
-      'You can change how Firefox works…',
-      '…or what it looks like',
-    ];
-    Array.from(rootNode.querySelectorAll('.HomePage-subheading'))
-      .map((el, index) => expect(el.textContent).toEqual(content[index]));
+  function render(props) {
+    const fakeDispatch = sinon.stub();
+
+    return shallow(
+      <HomeBase
```

Comment:
Hahaha I thought that too. ([For those not in the UK](http://www.homebase.co.uk/))